src/ripple/protocol/impl/STValidation.cpp

```diff
@@ -58,9 +58,14 @@ STValidation::validationFormat()
         {sfSigningPubKey,       soeREQUIRED},
         {sfSignature,           soeREQUIRED},
         {sfConsensusHash,       soeOPTIONAL},
+        // featureHardenedValidations
         {sfCookie,              soeDEFAULT},
         {sfValidatedHash,       soeOPTIONAL},
         {sfServerVersion,       soeOPTIONAL},
+        // featureXRPFees
+        {sfBaseFeeXRP,          soeOPTIONAL},
```

Comment:
Should these fields be, more accurately, named `sf[Whatever]Drops` instead? I don't mind it, as is, but I am curious what others think.

Reply:
I'd be fine with that. I just wanted to distinguish from the existing `sfBaseFee`, etc.

My thinking was that the data type of the field is `STAmount`, which has an `xrp()` function to return an `XRPAmount`, so for consistency "XRP" in the `SField` name makes sense.

Reply:
Done, but in a separate commit in case we decide that `sf[Whatever]XRP` is better.